### PR TITLE
refactor SettingsPage

### PR DESCRIPTION
**Pull request by Madara0330E**
made adjustments to the SettingsPage

### CODE REVIEW

File: luckydiamond/src/components/AsidebarComponent.vue
Comment (CkutlsGit) [inline luckydiamond/src/components/AsidebarComponent.vue:1]: A minor stylistic error is an unnecessary space on the 20th line. It's insignificant, but it's a good job.

File: luckydiamond/src/pages/SettingsPage.vue
Comment (CkutlsGit) [inline luckydiamond/src/pages/SettingsPage.vue:1]: A minor stylistic error is that the section with the "settings" class is not exactly positioned, I would also add a space between the 19th and 20th lines. As it is, the job is done perfectly.

File: luckydiamond/src/router/router.js
Comment (CkutlsGit) [inline luckydiamond/src/router/router.js:1]: Perfect job, but I would add a space between the 6th and 7th line.

**Review by CkutlsGit — CHANGES_REQUESTED** [review]
Good job. My verdict: **5-**
@Madara0330E

**Comment by danilt2000** [comment]
Good

**Pull request merged**